src/layers/metrics.rs

```diff
@@ -64,6 +64,25 @@ static METRIC_BYTES_WRITTEN_TOTAL: &str = "opendal_bytes_written_total";
 static LABEL_SERVICE: &str = "service";
 static LABEL_OPERATION: &str = "operation";
 
+/// update metrics on error results
+#[inline]
+fn inc_err_counter(result: std::io::Error, scheme: Scheme, op: Operation) -> std::io::Error {
```

Comment:
No. I don't like using abbr while not needed. Please use more readable naming.

And `Error` is not `result`, the naming is confusing.